src/box/space.c

```diff
@@ -325,6 +326,123 @@ space_unpin_defaults(struct space *space)
 	}
 }
 
+/**
+ * Set all the events associated with space: on_replace and before_replace,
+ * bound by name and by id. If argument on_recovery is true, recovery events
+ * are set instead of regular ones.
+ */
+static void
+space_set_events(struct space *space, bool on_recovery)
```

Comment:
We usually use 'is' or 'has' etc prefixes for boolean flags.

Reply:
Renamed to `is_on_recovery`.